2 - RTB/Documentazione interna/Verbali/24-02-11/24-02-11.typ

```diff
@@ -0,0 +1,106 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "11/02/24",
+  subTitle: "Meeting di retrospettiva e pianificazione",
+  docType: "verbale",
+  authors: (
+    "Oseliero Antonio",
+  ),
+  timeStart: "15:00",
+  timeEnd: "16:15",
+);
+
+= Ordine del giorno
+- Valutazione del progresso generale;
+- Analisi di retrospettiva;
+- Seconda parte RTB con #vardanega;
+- Pianificazione.
+
+== Valutazione del progresso generale
+
+Il meeting si avvia con con l'analisi dello sprint concluso. Nonostante si trattasse del primo sprint a seguito della sessione degli esami, la ripresa dei lavori è stata rapida (warm start), grazie anche alla scelta di non interrompere i lavori negli sprint precedenti ma di ridimensionare il carico di lavoro per far fronte agli impegni universitari, il corretto andamento dello sprint è supportato anche dalle metriche esposte sul cruscotto di qualità realizzato in Grafana.
+
+=== Revisione #ndp
+
+Sono stati revisionati la maggior parte dei capitoli del documento, anche se il lavoro non è ancora del tutto ultimato.\
+Rimangono le task relative a:
+- Rimuovere capitolo Processi di supporto/Documentazione;
+- Revisione ultimi capitoli rimasti.
+
+
+=== Revisione #adr
+
+Il gruppo si è impegnato a risolvere tutte le criticità riscontrate dal #cardin durante la prima parte di revisione RTB, precisamente la valutazione è stata:
+- Il glossario va riferito anche con il numero di versione del documento;
+- UC2: quali sono le informazioni che possono essere modificate?
+- UC3.1: quali sono i dati necessari alla creazione dello scaffale?
+- UC3.2: cosa può essere modificato di uno scaffale?
+- UC4.1: quali sono i dati necessari alla creazione di un bin?
+- UC4.2: cosa può essere modificato di un bin?
+- UC6: il database è interno all'applicazione o è possibile collegarsi a database esterni? Se siì, quali? Se invece è interno, allora il database non può essere un attore secondario (in ogni caso, la primitiva grafica scelta non esiste in UML 2);
+- UC6.1 è precondizione di UC6, non sua inclusione;
+- UC6.1 cosa si intende con "configurazione"? Quali informazioni?
+- UC7: cosa sono le API? Se sono API interne, allora non possono essere attore secondario. Inoltre, perché la funzionalità è una "richiesta"?
+- UC8 e seguenti: modificare "interrogazione" con "visualizzazione". Quali informazioni sono visualizzate?
+- UC11 va analizzato più in dettaglio. I requisiti di qualità mancano della creazione dei manuali richiesti;
+- VM1 e VM2 sono funzionali;
+- VM3 non è chiaro e quindi non verificabile;
+- Non è presente il tracciamento casi d'uso / requisiti.
+
+Il documento abbozza una struttura corretta, ma il livello di analisi raggiunto è spesso poco dettagliato e lacunoso. Inoltre, sono presenti alcuni errori di modellazione.\
+Tutti queste criticità sono state corrette durante lo sprint appena concluso e il documento è ora pronto per la revisione prima del PB.
```

Comment:
Da rivedere la parte "revisione prima del PB"... intendi forse che è pronto per essere revisionato prima dell'ingresso in PB?